Client/Frontend/Home/ActivityStreamPanel.swift

```diff
@@ -893,3 +911,15 @@ class ASHeaderView: UICollectionReusableView {
         fatalError("init(coder:) has not been implemented")
     }
 }
+
+open class PinnedSite: Site {
```

Comment:
A PinnedSite exists in order to tell regular topsites apart from Pinned ones in the view layer.